jni/src/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+# jnigen_native_build
```

Comment:
Maybe this should be

```
# Generated by `package:jnigen`.
# Special string to have it built by `dart run jni:setup`: "jnigen_native_build".
```

This way it's obvious why it is there, and users will not delete it.

Reply:
`package:jni` itself is not generated.

Can we shorten it so that it's obvious while not too sentence-y.

something similar to `// ignore_for_file:` directives

Reply:
How about `# jni_native_build (Build with jni:setup. Do not delete this line).

Reply:
sgtm